.gitignore

```diff
@@ -25,3 +25,4 @@ benchmark_outputs
 .mvn/timing.properties
 .editorconfig
 node_modules
+out/
```

Comment:
what's this? :)

Reply:
IntelliJ output directory. Maybe I should just change Intellij config to output somewhere else?

Reply:
Let's remove this as this seems irrelevant.